e3sm_to_cmip/cmor_handlers/handler.py

```diff
@@ -178,8 +178,8 @@ def cmorize(
         vars_to_filepaths: Dict[str, List[str]],
         tables_path: str,
         metadata_path: str,
+        cmor_log_dir: str,
         table: str | None = None,
-        logdir: str | None = None,
```

Comment:
Update order of parameters because `cmor_log_dir` is now a required parameter and should always have a value specified (even with the default).